extra-tools/index.md

```diff
@@ -28,15 +29,17 @@ extras(ctx, 'markdown', 'display', 'shortcutIcon');
 
 ## Markdown (`markdown`)
 
-You may have noticed most descriptions from annotations are parsed as Markdown. This is thanks to the Markdown filter from [sassdoc-extras][repo_extras] (using [Marked](https://github.com/chjj/marked)).
+You may have noticed most descriptions from annotations are parsed as Markdown. This is thanks to the Markdown filter from [sassdoc-extras][repo_extras] (using [Marked](https://github.com/chjj/marked)). This filter overrides the provided values.
 
-This filter overrides the provided values, so a value such as ``This is some `code`.`` would become ``This is some <code>code</code>``.
+For example, ``This is some `code`.`` becomes ``This is some <code>code</code>``.
 
 ## Display toggle (`display`)
 
-Depending on whether or not you want to display items based on their visibility (access level, `public` or `private`), the `display` filter might be able to help you. When used, it removes items that should not be displayed.
+The `display` filter allows you to display items based on their visibility (access level: `public` or `private`). This filter removes items that should not be displayed.
 
-To determine if an item should be displayed or not, this filter reads the `display.access` key from the context configuration (so make sure your theme defines one).
+The `display` filter uses the `display.access` key from the context configuration to determine if an item should be displayed or not.
+
+_**Note:** It's always best to define an access level._
```

Comment:
We have a predefined style for notes: `<p class="note note--info">`.  So maybe better to stick with it.